shuup/front/apps/customer_information/forms.py

```diff
@@ -29,6 +30,11 @@ def __init__(self, *args, **kwargs):
             self.fields[field].required = True
         self.initial["language"] = self.instance.language
 
+        field_properties = settings.SHUUP_PERSON_CONTACT_FIELD_PROPERTIES
+        for field, properties in field_properties.items():
+            for prop in properties:
+                setattr(self.fields[field], prop, properties[prop])
+
```

Comment:
I think it is OK to add this feature to 1.2, because it's very minor change and the implementation is so simple and straight forward. Also this code clearly does nothing if the setting is not modified from its default value (empty dict), so we should be fairly safe.